src/blog/another blog piece.md

```diff
@@ -0,0 +1,22 @@
+---
+title: This is another test post by Jared
+date: 2019-07-01
+tags:
+  - chocolate
+  - cake
+  - icecream
+---
+
+This is an intro!!!
+
+### Big fuckin heading
+
+ck before Graphical User Interfaces (or GUIs) were invented a [terminal](https://en.wikipedia.org/wiki/Computer_terminal) was a separate piece of hardware that let an operator interact with a computer that probably filled an entire separate room.
+
+## Smaller one
+
+Way back before Graphical User Interfaces (or GUIs) were invented a [terminal](https://en.wikipedia.org/wiki/Computer_terminal) was a separate piece of hardware that let an operator interact with a computer that probably filled an entire separate room.
+
+# Even smaller one
```

Comment:
You have your headings (understandably) backwards. A single hash creates an `h1` (the main page title of which there should only ever be one on the page). Two hashes is an `h2` (second level heading), which is for naming top-level sections. Three hashes is a third level heading (for sub-sections underneath an `h2`), and so on down to `h6`

Reply:
Ohhhh that makes a lot of sense.